packages/merge/src/modules/merge/sagas.js

```diff
@@ -53,20 +66,13 @@ export function* getMergeBody() {
     }))
   }
 
-  const mergeRelations = []
-  sourceData.relations.forEach(relation => {
-    const isSelected = selected.multipleAll[relation.relationName].includes(relation.entityKey)
-    if (isSelected || relation.entityKey === selected.targetEntity) {
-      mergeRelations.push({
-        relationName: relation.relationName,
-        sourceKey: relation.entityKey,
-        selected: isSelected
-      })
-    }
-  })
+  const mergeRelations = _uniq(sourceData.relations.map(r => r.relationName)).map(relationName => ({
```

Comment:
2 maps necessary? Couldn't you just remove the first map and use "r.relationName" in the second one?

Reply:
the first map is inside the _uniq method to remove duplicates. merging into one map is not possible

Reply:
sorry ...my fault :)